internal/events/event.go

```diff
@@ -24,6 +26,8 @@ var DefaultEventCategories = []string{
 	models.EventCategoryUser,
 }
 
+var defaultEventLimit int64 = 5000
```

Comment:
Why default to 5000?

Reply:
There is no good reason, I coordinated with the UI that in this case we will set `Limit` to some default, I chose 5000 as it is usually large enough to retrieve all cluster's events, but not too large

Reply:
Wouldn't UI use limit in the future? Why do we need to break compatibility with the past and limit requests that are not using limit parameters?
AFAIK only `assisted-events-scrape` uses this (apart from UI), and it takes only the first 2000 results reverse ordered by event_time, but I still wouldn't break compatibility.

Reply:
If no `Limit` is specified (nil), it is set to -1 (unlimited). If `Limit` is set improperly (smaller than -1 or 0), I set it to default.

Reply:
I see, sorry I didn't notice that, got tricked by the naming.